config/environments/development.rb

```diff
@@ -79,8 +79,8 @@ def yarn_integrity_enabled?
     address: "smtp.gmail.com",
     port: "587",
     enable_starttls_auto: true,
-    user_name: '<%= ENV["DEVELOPMENT_EMAIL_USERNAME"] %>',
-    password: '<%= ENV["DEVELOPMENT_EMAIL_PASSWORD"] %>',
+    user_name: '<%= ApplicationConfig["DEVELOPMENT_EMAIL_USERNAME"] %>',
```

Comment:
no need to change this to `ApplicationConfig` since that would make it mandatory.